collect_app/src/main/java/org/odk/collect/android/activities/FormDownloadList.java

```diff
@@ -341,8 +331,9 @@ private void downloadFormList() {
     @Override
     protected void onSaveInstanceState(Bundle outState) {
         super.onSaveInstanceState(outState);
-        outState.putBoolean(BUNDLE_TOGGLED_KEY, mToggled);
-        outState.putInt(BUNDLE_SELECTED_COUNT, selectedItemCount());
+        // TODO: Is it safe to remove BUNDLE_TOGGLED_KEY ?
```

Comment:
I mean is it safe to remove the writing out and reading back in of the value. 

Though I don't see a need to keep the constant around either if it is not used.

Your opinion?